layouts/partials/widgets/social.html

```diff
@@ -42,6 +42,14 @@ <h4 class="widget-social__title widget__title">Social</h4>
 			</a>
 		</div>
 		{{- end }}
+		{{- with .Site.Params.widgets.social_google_plus }}
+		<div class="widget-social__item widget__item">
+			<a class="widget-social__link widget__link btn" title="Google Plus" href="https://plus.google.com/{{ . }}">
+				<svg class="widget-social__link-icon icon-google-plus" viewBox="0 0 2304 1792" width="24" height="24" xmlns="http://www.w3.org/2000/svg" fill="#fff"><path d="M1437 913q0 208-87 370.5t-248 254-369 91.5q-149 0-285-58t-234-156-156-234-58-285 58-285 156-234 234-156 285-58q286 0 491 192l-199 191q-117-113-292-113-123 0-227.5 62t-165.5 168.5-61 232.5 61 232.5 165.5 168.5 227.5 62q83 0 152.5-23t114.5-57.5 78.5-78.5 49-83 21.5-74h-416v-252h692q12 63 12 122zm867-122v210h-209v209h-210v-209h-209v-210h209v-209h210v209h209z"/></svg>
```

Comment:
`icon-google-plus` not compatible with BEM notation

Reply:
@Vimux , oh! It's my settings. Of course, I'll change this.

Reply:
@Vimux , could you give me an advice for the naming?
1. `icon-google_plus`
1. `icon-googleplus`